docs/conf.py

```diff
@@ -22,7 +24,8 @@
 
 project = "Bayesian Tracker (btrack) 🔬💻"
 author = "Alan R Lowe"
-copyright = f"2017-{datetime.date.today().year}, {author}"
+year = datetime.datetime.now(tz=pytz.timezone("GMT")).year
```

Comment:
Is `sphinx` going to be okay with an additional field `year`?

Reply:
Also, I noticed that the start year in the C++ files in 2014

Reply:
Well-spotted! Should I update the start year for the copyright?

Reply:
> Is sphinx going to be okay with an additional field year?

I think it should be fine as it's not a `sphinx` keyword so it should be ignored

Reply:
I've set the start year to 2014

Reply:
Okay cool